README.md

```diff
@@ -15,13 +15,21 @@ FedScale requires Python 3.7 or better.
 We use [Anaconda](https://www.anaconda.com/products/distribution#download-section) environments to manage its dependencies.
 FedScale also requires `FEDSCALE_HOME` environment variable set to the path of the FedScale installation directory.
 
-Once you have Anaconda installed, here are the instructions assuming that the current directory is where you have cloned FedScale to.
+Once you have Anaconda installed, here are the instructions.
 
 ```
-export FEDSCALE_HOME=$PWD
+git clone https://github.com/SymbioticLab/FedScale.git
```

Comment:
Too many random blank lines. Please replace all commands to 

```
git clone https://github.com/SymbioticLab/FedScale.git
cd FedScale

# Please replace ~/.bashrc with ~/.bash_profile for MacOS
echo export FEDSCALE_HOME=$(pwd) >> ~/.bashrc 
conda init bash
. ~/.bashrc

conda env create -f environment.yml
conda activate fedscale
pip install -e .
```

Reply:
Done

Reply:
Is this the best way to do it? It just makes the instructions long and ugly to look at in the first page?

Reply:
Also update `install.sh` with equivalent info and comments.